go-controller/pkg/ovn/egressip.go

```diff
@@ -1888,7 +1888,7 @@ func (e *egressIPZoneController) deleteEgressIPStatusSetup(name string, status e
 	if err == nil {
 		eIPIP := net.ParseIP(status.EgressIP)
 		if eIPConfig, err := util.GetNodeEIPConfig(eNode); err != nil {
-			klog.Warningf("Failed to get Egress IP config from node annotation: %v", name, eIPIP, err)
+			klog.Warningf("Failed to get Egress IP config from node annotation: %s, %v, %v", name, eIPIP, err)
```

Comment:
Too cryptic. I think the most important bit to log here is the node name which is not included. I would just have that and the error

Reply:
ack let me remove eIPIP and give nodeName instead. because name already has EIP name

Reply:
I think name here is the egress IP name and not the node name. so you would have to use `status.Node`

Reply:
done.